website/pages/docs/references/eslint.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: ESLint Plugin
+description: ESLint plugin for Panda CSS 
+---
+
+# ESLint Plugin
+
+ESLint plugin for Panda CSS: `@pandacss/eslint-plugin`.
+
+## Installation
+
+<Tabs items={['pnpm', 'npm', 'yarn', 'bun']} variant="code">
+  <Tab>
+    ```bash
+    pnpm add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm install -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    bun add -D @pandacss/eslint-plugin
+    ```
+  </Tab>
+</Tabs>
+
+## Rules
+
+### `@pandacss/eslint-plugin/file-not-included`
+
+Prevent panda usage in files not convered by panda `include` config.
+
+### `@pandacss/eslint-plugin/no-debug`
+
+Prevent shipping the debug attribute to production.
+
+### `@pandacss/eslint-plugin/no-shorthand-prop`
+
+Discourage the use of shorthand properties. Prefer to use longhand css property instead.
+
+### `@pandacss/eslint-plugin/no-unsafe-token-fn-usage`
+
+Ensure user does not use token fn in places they could just use the raw design token.
+
+### `@pandacss/eslint-plugin/prefer-atomic-properties`
+
+Prefer atomic properties over shorthand properties.
+
+## Settings
+
+### `configPath`
+
+You can tell `eslint` to use a custom panda config file by setting the `configPath` option in your `.eslintrc.js` file.
+
+By default we find the nearest panda config to the linted file.
+
+```json
+{
+  "plugins": [
+    "@pandacss/eslint-plugin"
+  ],
+  "settings": {
+    "pandacss/configPath": "/ABSOLUTE-PATH-TO/panda.config.js"
+  }
+}
```

Comment:
[Bug] Hardcoded paths in ESLint aren't a common practice because they're often not predictable. E.g. CI machines will have different absolute paths than local configs.

The occasional need to do something like `path.join` is why a lot of folks tend to write ESLint configs as `.js`:

```suggestion
```js
const path = require("path");

module.exports = {
  plugins: [
    "@pandacss/eslint-plugin"
  ],
  settings: {
    "pandacss/configPath": path.join("PATH-TO/panda.config.js")
  }
};
```

Reply:
@JoshuaKGoldberg This would be a `eslint.config.js` config right?

Reply:
No, this code block is for the current/legacy-style `.eslintrc.(c)js`. That's what I personally use.